newscript.go

```diff
@@ -125,15 +147,15 @@ func blankBootstick() error {
 			break
 		}
 	}
-	fmt.Printf("Running dd to put the new image onto the desired location. \n")
+	fmt.Printf("Running cp to put the new image onto the desired location. \n")
```

Comment:
So this is weird. Here you do a cp and down below you do a dd. Which one did you want to do? I don't understand this.